### ISSUE TITLE
Accessibility - Colour contrast

### ISSUE THREAD

**Issue by Rhiana** [strategicdata/websurvey.com.au#1]
There are a few items on the site where the colour contrast is not high enough to meet AA accessibility guidelines.
Please advise if it's ok for me to alter the shades of the colours a little to meet the contrast guidelines.

* Why WS panel, 3.87:1 very close, the headings are ok but not the smaller text
![screen shot 2015-11-27 at 2 04 36 pm](https://cloud.githubusercontent.com/assets/5127756/11433362/146b0a12-9510-11e5-9f96-af9768b1d482.png)

* Navigation items on hover (contact), 2.48:1 fail
![screen shot 2015-11-27 at 2 04 50 pm](https://cloud.githubusercontent.com/assets/5127756/11433376/37989b80-9510-11e5-8e94-290378c0c6a2.png)

- Subscribe button. 2.32:1 fail, disappears completely in high contrast mode
![screen shot 2015-11-27 at 2 05 00 pm](https://cloud.githubusercontent.com/assets/5127756/11433381/4bbff6d0-9510-11e5-9e18-ae04c683abc9.png)

* Subscribe alert text, 1.58:1 fail
![screen shot 2015-11-27 at 2 06 05 pm](https://cloud.githubusercontent.com/assets/5127756/11433388/66a3c814-9510-11e5-8400-1601d05618a7.png)

**Comment by michellenSD** [comment]
Sounds fine to me @Rhiana, thanks!

**Comment by Rhiana** [comment]
I have updated the colour contrast for accessibility levels by making colours darker or lighter trying to preserve the theme as much as possible. I have also made sure that any on hover changes also happened on focus for keyboard accessibility.

The blue is slightly darker
![screen shot 2015-12-02 at 9 50 49 am](https://cloud.githubusercontent.com/assets/5127756/11517223/770c5fe8-98dc-11e5-8764-2cf2345b38c3.png)


The hover/focus is white but with an opacity of 80% so it a bit lighter
![screen shot 2015-12-02 at 10 03 29 am](https://cloud.githubusercontent.com/assets/5127756/11517192/5d1d4390-98dc-11e5-9f31-a3fbee00f24a.png)

The subscribe button I made the text darker and the background lighter. (I also made the text in the input darker as before it was white on white)
![screen shot 2015-12-02 at 10 18 59 am](https://cloud.githubusercontent.com/assets/5127756/11517520/22fb1230-98de-11e5-8b88-9c400553d2bc.png)

The confirmation text is also lighter
![screen shot 2015-12-02 at 10 20 26 am](https://cloud.githubusercontent.com/assets/5127756/11517559/58e83bca-98de-11e5-828d-9aac91fa4f02.png)
